finesse/gui/measure_script/script_view.py

```diff
@@ -6,7 +6,7 @@
 from PySide6.QtWidgets import QFileDialog, QGridLayout, QGroupBox, QPushButton
 
 from ...config import DEFAULT_SCRIPT_PATH, STEPPER_MOTOR_TOPIC
-from ...em27_status import EM27Status
+from ...em27_info import EM27Status
 from ...event_counter import EventCounter
 from ...settings import settings
 from ..path_widget import OpenFileWidget
```

Comment:
We should have some consistency on importing internal stuff, either using relative imports like here or absolute ones, as you have done in other cases.

Reply:
True! I've opened an issue to do this project-wide (#378).